style/ruby/README.md

```diff
@@ -25,6 +24,7 @@ Ruby
 * Prefer `&:method_name` to `{ |item| item.method_name }` for simple method
   calls.
 * Prefer `if` over `unless`.
+* Prefer explicit, early `return`s over `if`-`else`.
```

Comment:
I think this should be "Prefer explicit, early `return`s over `if`-`end`". If the conditional has an `else` that is meaningful (i.e. something other than `nil`), then I think it's preferable that it remain a conditional instead of breaking it up into an early return. 

For example,

```
def foo(bar)
  if bar.present?
    bar.to_s + " badabam"
  else
    "No badabam here"
  end
end
```
is preferable (IMO) to 

```
def foo(bar)
  return "No badabam here" unless bar.present?

  bar.to_s + " badabam"
end
```

Whereas if the `else` clause isn't present, then it makes sense to have an early return. The reason I think it's clearer not to use early return in this example is because it's harder for me to identify and parse logic and meaningful values buried in the early return one-liner.